client/app/bundles/course/assessment/submission/constants.js

```diff
@@ -28,6 +28,15 @@ export const TestCaseTypes = {
   Evaluation: 'evaluation_test',
 };
 
+export const selectedUserType = {
+  my_students: 'MY STUDENTS',
+  my_students_w_phantom: 'MY STUDENTS incl. phantom',
```

Comment:
Can the casing be standardised? Especially if it's gonna to be shown to the end users.

Reply:
Noted